asllib/doc/Specifications.tex

```diff
@@ -275,6 +290,58 @@ \subsubsection{Formally}
 \end{mathpar}
 \CodeSubsection{\TypeCheckMutuallyRecBegin}{\TypeCheckMutuallyRecEnd}{../Typing.ml}
 
+\subsubsection{TypingRule.TypeCheckGlobalPragmas\label{sec:TypingRule.TypeCheckGlobalPragmas}}
```

Comment:
This is correct, but I think it would be simpler to have a function to check a single global pragma, e.g., `check_global_pragma` (TypingRUle.CheckGlobalPragma) and then apply that to the entire list as a premise in checking a specifications, as opposed to explicitly processing the list here. Also, the term `annotate` should be `checked`, as annotate is used for "returns a typed AST node".